docs/style-guide.md

```diff
@@ -0,0 +1,62 @@
+# Documentation Style Guide
+
+```{note}
+This document is still a work in progress
+```
+
+This is a style guide for the ndindex documentation.
+
+## English Conventions
+
+- Use American English spelling for all documentation.
```

Comment:
My centre favours the Brits :rofl:

Reply:
It's important that the documentation not have any spelling errors :)

FWIW the only reason I created the style guide was to write down the "indices" vs. "indexes" thing, but I figured there were other unwritten conventions so I should write them down too.